packages/tools/eslint-config/profile/next.js

```diff
@@ -2,4 +2,7 @@ module.exports = {
   root: true,
   extends: ['./react', 'next/core-web-vitals'],
   plugins: ['import', 'simple-import-sort'],
+  rules: {
+    'react/prop-types': 'off',
+  },
```

Comment:
Do we need to remove this? It should check for statically typed in flow or typescript and then the rule should succeed: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/prop-types.md#:~:text=Note%3A%20You%20can%20provide%20types%20in%20runtime%20types%20using%20PropTypes%20and/or%20statically%20using%20TypeScript%20or%20Flow.%20This%20rule%20will%20validate%20your%20prop%20types%20regardless%20of%20how%20you%20define%20them.